src/main/kotlin/apply/application/MissionService.kt

```diff
@@ -55,10 +53,8 @@ class MissionService(
         return request.id == 0L || !missionRepository.existsById(request.id)
     }
 
-    fun getDataById(id: Long): MissionData {
-        val mission = missionRepository.getById(id)
-        val evaluation = evaluationRepository.getById(mission.evaluationId)
-        return MissionData(mission, evaluation)
+    fun getById(id: Long): MissionResponse {
+        return missionRepository.getById(id).let(::MissionResponse)
```

Comment:
`let`부분에서 한 줄에 점 두 개 쓰고 있는데 의도하신 건가요?

Reply:
원칙은 `.` 문자나 `?.` 연산자가 올 때 줄 바꿈이 필요하기 때문에 아래 코드가 맞지만 한 줄로 표현하는 것이 합리적이라고 생각했어요. @asebn1 은 어떻게 생각하시나요?
https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls

```suggestion
    fun getById(id: Long): MissionResponse {
        return missionRepository
            .getById(id)
            .let(::MissionResponse)
```